domains/ijas.json

```
@@ -0,0 +1,10 @@
+{
+  "subdomain": "ijas",
+  "owner": {
+    "email": "ijas.n.ijas@gmail.com"
```

github username is required
```suggestion
    "username": "ijasxyz",
    "email": "ijas.n.ijas@gmail.com"
```